examples/lock-app/efr32/src/LockManager.cpp

```diff
@@ -294,15 +302,19 @@ bool LockManager::SetUser(uint16_t userIndex, chip::FabricIndex creator, chip::F
     userInStorage.credentials = chip::Span<const DlCredential>(mCredentials.Get(), totalCredentials);
```

Comment:
And this has the same issue.  Please check all places `mCredentials` is used.

Reply:
Fixed mCredentials